src/sourmash/minhash.py

```diff
@@ -684,6 +684,22 @@ def to_frozen(self):
         new_mh.__class__ = FrozenMinHash
         return new_mh
 
+    def inflate(self, from_mh):  
+        if (self.track_abundance == False) and (from_mh.track_abundance == True):
+            hashes = set(self.hashes)
+            orig_abunds = from_mh.hashes
+            abunds = { h: orig_abunds[h] for h in hashes }
+
+            abund_query_mh = from_mh.copy_and_clear()
+
+            abund_query_mh.downsample(scaled=self.scaled)
+            abund_query_mh.set_abundances(abunds)
+            self.minhash = abund_query_mh
+
+            return abund_query_mh
+        else:
+            raise ValueError('value of track_abundance for self and from_mh should be true') 
```

Comment:
is this error message still accurate?

Reply:
No, I'll fix that. It should be `"value of track_abundance for self should be false and from_mh should be true"`.